storage/s3/src/main/kotlin/S3StorageProvider.kt

```diff
@@ -51,33 +50,31 @@ class S3StorageProvider(
     /**
      * Retrieve the data associated with the given [key] from the S3 bucket.
      */
-    override fun read(key: Key): StorageEntry = runBlocking {
-        s3Client.getObject(
-            GetObjectRequest {
-                bucket = bucketName
-                this.key = key.key
-            }
-        ) { response ->
-            val data = checkNotNull(response.body) { "No data found for ${key.key}." }
-            val contentType = response.contentType
-            val tempFile = createTempFile()
-
-            runCatching {
-                data.writeToFile(tempFile)
-                StorageEntry.create(tempFile.toFile(), contentType)
-            }.onFailure { tempFile.deleteExisting() }.getOrThrow()
+    override suspend fun read(key: Key): StorageEntry = s3Client.getObject(
```

Comment:
Not sure about the formatting: I think for lambdas it was just ok to start on this line, but I'm unsure for function calls. Any guideline here, @mnonnenmacher?

Reply:
@sschuberth Are you fine with merging the PR ?

Reply:
Sure, I've just put it into the merge queue.